src/test/java/model/report/TestTeacherReport.java

```diff
@@ -0,0 +1,56 @@
+package model.report;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.Test;
+
+public class TestTeacherReport {
+  private final TeacherReport emptyTeacherReport = new TeacherReport();
+  private final TeacherReport teacherReport = new TeacherReport("xd", "xd", "xd", 40);
+
+  @Test
+  public void testGetFirebaseId() {
```

Comment:
Name of the test should be descriptive enough to suggest what is expected and the circumstances of this process without scrutinising what is happening in the method.
The way I like to most is "should ... When ..."
Also every test case should be different assertion (I mean @Test annotation)
So i.e.
`@Test
public void shouldReturnNullWhenTeacherHasEmptyId {...}` etc.

Reply:
czemu Ty piszesz komentarze po angielsku?